Tests/FrisbeeTests/Entities/FrisbeeErrorTests.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import Frisbee
+import XCTest
+
+class FrisbeeErrorTests: XCTestCase {
+    func testEquality() {
+        let all = FrisbeeError.all(error: SomeError.some)
+
+        // asserts diagonal equality
+        zip(all, all).forEach { lhs, rhs in
+            XCTAssertEqual(lhs, rhs)
+        }
+    }
+
+    func testInequality() {
+        let all = FrisbeeError.all(error: SomeError.some).enumerated()
+        all.extensiveCombine(all).forEach { lhs, rhs in
+            if lhs.offset == rhs.offset { //diagonal cases
+                XCTAssertEqual(lhs.element, rhs.element)
+            } else {
+                XCTAssertNotEqual(lhs.element, rhs.element)
+            }
+        }
+    }
+}
+
+extension FrisbeeError {
+    static func all(error: Error) -> [FrisbeeError] {
```

Comment:
To simplify this function, maybe `error` could be just some string. And maybe `SomeError` doesn't need to exist 🙂